src/Components/Card/index.jsx

```diff
@@ -56,7 +57,7 @@ const Card = (props) => {
   }
 
   return (
-    < div className={classNames('card', { 'card_is_red': props.card.chooseСardFlag, 'card_is_green': !props.card.chooseСardFlag })}>
+    <Aux>
```

Comment:
If you're using Aux hoc just to wrap up components and have one node to return - just use `React.Fragment` hoc which works the same way and have the only function - to group the wrapped components.